packages/ssr/src/lib.rs

```diff
@@ -38,6 +38,7 @@ impl SsrRenderer {
     }
 }
 
+#[allow(clippy::needless_lifetimes)]
```

Comment:
I believe the below could be 
```rust
pub fn render_lazy(f: LazyNodes<'_, '_>) -> String {
```
But since we name the lifetimes in the Safety comment below, I allowed it.